src/containers/task-group.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Task from '../components/task';
+
+class TaskGroup extends Component {
+
+  processTaskList(list, givenMonth, givenType) {
+    let resultsArray = [];
+    const loop = list.map( (item) => {
```

Comment:
If you're not using the return value, you can just do `list.forEach` instead of `list.map` (otherwise identical - give it the lambda, etc.).

That said, I think you can actually do better here using `let resultsArray = list.filter(item => item.month === givenMonth && item.type === givenType)`.